install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3047,6 +3047,12 @@ clustermesh:
       # @schema
       # -- The internalTrafficPolicy of service used for apiserver access.
       internalTrafficPolicy: Cluster
+      # -- Defines when to enable session affinity.
+      # Possible values:
+      #  - "HAOnly" (default) only enable session affinity for deployments with more than 1 replica.
+      #  - "Always"
+      #  - "Never"
```

Comment:
I'd suggest to expand a bit the comments to clarify what the different options mean. In particular, that the `Never` option can be configured to enable HA in combination with LBs that do not support session affinity, and that even though it does not introduce correctness issues, it may lead to slightly degraded performance due to more frequent reconnections.